### PR TITLE
Create list and detail view

### PR DESCRIPTION
**Pull request by Karthikeyantestpress**

### CODE REVIEW

File: mysite/blog/templates/blog/base.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{% load static %}
```
Comment (waranlogesh) [inline mysite/blog/templates/blog/base.html:1]: Use formatter for html files. It is unreadable without indentation.

File: mysite/blog/tests/test_modelmixintestcase.py [hunk 2]
```diff
@@ -15,16 +16,30 @@ def setUp(self):
...
+        self.post_list_url = reverse("blog:post_list")
+        self.post_detail_url = reverse(
+            "blog:post_detail",
+            args=[
+                self.published_post.publish.year,
+                self.published_post.publish.month,
+                self.published_post.publish.day,
+                self.published_post.slug,
+            ],
         )
```
Comment (waranlogesh) [inline mysite/blog/tests/test_modelmixintestcase.py]: Use it directly in the testcases

**Review by waranlogesh — CHANGES_REQUESTED** [review]

File: mysite/blog/templates/blog/base.html (hunk 1 above)
Comment (Karthikeyantestpress) [reply mysite/blog/templates/blog/base.html:1]: Formatted the HTML files

File: mysite/blog/tests/test_modelmixintestcase.py (hunk 2 above)
Comment (Karthikeyantestpress) [reply mysite/blog/tests/test_modelmixintestcase.py]: Made changes as per suggestion

**Review by waranlogesh — APPROVED** [review]

**Pull request merged**